.vscodeignore

```diff
@@ -1,17 +1,45 @@
-.vscode/**
-.vscode-test/**
-out/test/**
-out/**/*.map
-out/**/*.ts
-src/**
+.vscode
+.vscode-test
+!node_modules/keytar
+!node_modules/getmac
+!node_modules/editions
+!node_modules/errlop
+!node_modules/extract-opts
+!node_modules/typechecker
+!node_modules/eachr
+!node_modules/getos
+!node_modules/node-usb-native/native
+!node_modules/usb-detection/build/Release/detection.node
+node_modules/usb-detection/doc
+node_modules/usb-detection/test
+!node_modules/usb-detection/*.*
+!node_modules/eventemitter2
+!node_modules/bindings
+!node_modules/file-uri-to-path
+!node_modules/@serialport/bindings
+!node_modules/@serialport/binding-abstract
+!node_modules/@serialport/parser-readline
+!node_modules/@serialport/parser-delimiter
+!node_modules/debug
+!node_modules/ms
+node_modules
+out
+src
+vendor
```

Comment:
Have we removed vendor folder with the change of usb-detection?

Reply:
vendor folder has already been deleted by dilin's change to node-usb-native module.

Reply:
Then maybe we should remove the `vendor` folder here?